packages/core/src/types.ts

```diff
@@ -1316,8 +1317,8 @@ export type ActorRefFrom<
     }
   : T extends Promise<infer U>
   ? ActorRef<never, U>
-  : T extends Behavior<infer TEvent, infer TEmitted>
-  ? ActorRef<TEvent, TEmitted>
+  : T extends Behavior<infer TEvent1, infer TEmitted>
```

Comment:
If you have switched to using the "ordinal" suffix here then I would expect to see `TEvent0` in the `StateMachine` match above, OTOH - maybe it's more ergonomic to not use those suffixes at all? do they improve readability for you?

Reply:
TypeScript warns when I use the same names for type parameters.

Reply:
6a197fcacd22876770c350ead150ec8e1d9d13cc